s3torchconnector/src/s3torchconnector/__init__.py

```diff
@@ -1,6 +1,6 @@
 #  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 #  // SPDX-License-Identifier: BSD
-from s3torchconnectorclient import S3Exception
+from s3torchconnectorclient import S3Exception, enable_debug_logging
```

Comment:
Thinking out loud - perhaps we also need a wrapper on the Python side of things that takes in a logging config and a log level and configures the logging module like below, but with the passed in parameters:
```
logging.basicConfig(
    format="%(levelname)s %(name)s %(asctime)-15s %(filename)s:%(lineno)d %(message)s"
)
logging.getLogger().setLevel(LOG_TRACE)
```